pip/pip-409.md

```diff
@@ -33,25 +33,88 @@ retry/dead letter topic.
 
 ## Design & Implementation Details
 
-- Add two new configurations in `DeadLetterPolicy`:
+- Add `ProducerBuilderContext` and `ProducerBuilderCustomizer` interface:
```

Comment:
Update the naming also in the PIP document since it's being maintained in this PR.

Reply:
fixed.